src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -660,7 +670,8 @@ public static void refreshSession() {
 
     boolean shouldResetCounters = false;
     boolean shouldResetGlobalCounters = false;
-    // Assume if rollover has changed by an hour, it is a new rollover. Time varies slightly between
+    // Assume if rollover has changed by an hour, it is a new rollover. Time varies
+    // slightly between
     // servers by a few seconds.
```

Comment:
Can you reflow this comment? Now you have two short lines next to each other.

ditto below. Do you have your column limit set to 80 chars? 100 is the standard in Google Java Style.

Reply:
oh i think this was an auto format in my editor, i'll revert them